concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+ID	NUMBER	ENGLISH	CHINESE	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Zhou-2020-350-1	1	all	全部	98	ALL
+Zhou-2020-350-2	2	almost	快要		
+Zhou-2020-350-3	3	ant	螞蟻	587	ANT
+Zhou-2020-350-4	4	ashes	灰	646	ASH
+Zhou-2020-350-5	5	at first	先	2915	AT FIRST
+Zhou-2020-350-6	6	bad	壞	1292	BAD
+Zhou-2020-350-7	7	bamboo	竹子	1927	BAMBOO
+Zhou-2020-350-8	8	bamboo mat	竹蓆	3240	BAMBOO MAT
+Zhou-2020-350-9	9	beast	獸	958	WILD ANIMAL
+Zhou-2020-350-10	10	bed	牀	1663	BED
+Zhou-2020-350-11	11	belly	肚子	1251	BELLY
+Zhou-2020-350-12	12	belt	腰帶	772	BELT
+Zhou-2020-350-13	13	big	大	1202	BIG
+Zhou-2020-350-14	14	bitter	苦	887	BITTER
+Zhou-2020-350-15	15	black	黑	163	BLACK
+Zhou-2020-350-16	16	blood	血	946	BLOOD
+Zhou-2020-350-17	17	boat	船	1844	BOAT
+Zhou-2020-350-18	18	boiling water	開（水）	3362	BOILED WATER
+Zhou-2020-350-19	19	bottom	底下	834	BUTTOCKS
+Zhou-2020-350-20	20	bovine	牛	3044	BOVINE
+Zhou-2020-350-21	21	bowl	碗	1530	BOWL
+Zhou-2020-350-22	22	bran	糠	2362	BRAN
+Zhou-2020-350-23	23	bright	亮	679	BRIGHT
+Zhou-2020-350-24	24	bug	蟲	1027	BUG
+Zhou-2020-350-25	25	buy	買	1869	BUY
+Zhou-2020-350-26	26	cat	貓	1208	CAT
+Zhou-2020-350-27	27	cave	洞	1155	CAVE
+Zhou-2020-350-28	28	cedar tree	杉樹		
+Zhou-2020-350-29	29	chilly	涼		
+Zhou-2020-350-30	30	Chinese	漢	3373	HAN CHINESE
+Zhou-2020-350-31	31	chopsticks	筷子	1987	CHOPSTICKS
+Zhou-2020-350-32	32	cl. for animals	（一）頭（牛）		
+Zhou-2020-350-33	33	clf. for clothes	件		
+Zhou-2020-350-34	34	clf. for general objects	個		
+Zhou-2020-350-35	35	cloth	布	1616	CLOTH
+Zhou-2020-350-36	36	clothes	衣服	1895	CLOTHES
+Zhou-2020-350-37	37	cold	冷	1287	COLD
+Zhou-2020-350-38	38	comb	梳子	1475	COMB
+Zhou-2020-350-39	39	crow	烏鴉	2074	CROW (VERB)
```

Comment:
This should be mapped to the noun 913 CROW.

Reply:
Yes, or Raven, they do not really distinguish.